GVRf/Framework/jni/objects/bounding_volume.cpp

```diff
@@ -171,6 +171,35 @@ void BoundingVolume::transform(const BoundingVolume &in_volume, glm::mat4 matrix
     expand(transformed_center, radius);
 }
 
+//Transform existing bounding volume by matrix.
+void BoundingVolume::transform(const glm::mat4 &matrix) {
```

Comment:
maybe it's late and i'm tired, but i'm not fully following the logic in here.  perhaps add more comments?